app/src/main/java/org/fossasia/openevent/general/utils/Utils.kt

```diff
@@ -139,6 +141,56 @@ object Utils {
         }
     }
 
+    fun getNextDate(): String {
+        val calendar = Calendar.getInstance()
+        calendar.add(Calendar.DATE, 1)
+        return getSimpleFormattedDate(calendar.time)
+    }
+
+    fun getNextToNextDate(): String {
+        val calendar = Calendar.getInstance()
+        calendar.add(Calendar.DATE, 2)
+        return getSimpleFormattedDate(calendar.time)
+    }
+
+    fun getWeekendDate(): String {
+        val calendar = Calendar.getInstance()
+        val today = calendar.get(Calendar.DAY_OF_WEEK)
+        if (today != Calendar.SATURDAY) {
+            val offset = Calendar.SATURDAY - today
+            calendar.add(Calendar.DATE, offset)
+        }
+        return getSimpleFormattedDate(calendar.time)
+    }
+
+    fun getNextToWeekendDate(): String {
+        val calendar = Calendar.getInstance()
+        val today = calendar.get(Calendar.DAY_OF_WEEK)
+        if (today != Calendar.SATURDAY) {
+            val offset = Calendar.SATURDAY - today
+            calendar.add(Calendar.DATE, offset)
+        }
+        calendar.add(Calendar.DATE, 1)
+        return getSimpleFormattedDate(calendar.time)
+    }
+
+    fun getNextMonth(): String {
+        val calendar = Calendar.getInstance()
+        val today = calendar.get(Calendar.DAY_OF_MONTH)
+        val offset = 30 - today
+        calendar.add(Calendar.DATE, offset)
+        return getSimpleFormattedDate(calendar.time)
+    }
+
+    fun getNextToNextMonth(): String {
+        val calendar = Calendar.getInstance()
+        val today = calendar.get(Calendar.DAY_OF_MONTH)
+        val offset = 30 - today
+        calendar.add(Calendar.DATE, offset)
+        calendar.add(Calendar.MONTH, 1)
+        return getSimpleFormattedDate(calendar.time)
+    }
```

Comment:
Move to DateTimeUtils

Reply:
There is no such class/file, do you want me to create one or should do you want me to move to EventUtils??

Reply:
Create

Reply:
okay working on it

Reply:
done